File: app/Models/TheAllPurposeProgrammer/HelloWorld/java/HelloWorld.java
public class HelloWorld {

	public static void main(String[] args) {
		String first_name = "Jacob";
		String last_name = "Landry";

		System.out.println(hello(first_name, last_name));
	}

	public static String hello(String first_name, String last_name) {
		return "Hello " + first_name + " " + last_name + ", how are you doing today?";
	}

}

File: TheAllPurposeProgrammer/HelloWorld/cpp/HelloWorld.cpp
#include <iostream>
#include <string>

using namespace std;

string hello(string first_name, string last_name)
{
    return "Hello " + first_name + " " + last_name + ", how are you doing today?";
}

int main()
{
    string first_name = "Jacob";
    string last_name = "Landry";


    std::cout << hello(first_name, last_name);
}